src/main/java/zmq/Ctx.java

```diff
@@ -127,6 +134,13 @@ private void destroy() throws IOException
             it.close();
         }
 
+        for (Selector selector : selectors) {
+            if (selector != null) {
```

Comment:
If there should never be a null entry in the list, then another approach is to enforce the null check when a selector is added and throw NPE at that time.